### PR TITLE
Update navigation panel VC-1718

### PR DESCRIPTION
**Pull request by konutis**

### CODE REVIEW

File: public/components/navigationSlider/navigationSlider.js [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+    this.navigationSliderRef.current.addEventListener('scroll', this.listenForScroll)
+    this.handleResize()
+  }
+
+  componentWillUnmount () {
+    this.resizeObserver.unobserve(this.navigationContainerRef.current)
+    this.navigationSliderRef.current.remove('scroll', this.listenForScroll)
+  }
+
+  listenForScroll () {
```
Comment (pavelthq) [inline public/components/navigationSlider/navigationSlider.js:46]: event listeners must be prefixed "handle" https://medium.com/javascript-in-plain-english/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364

File: public/sources/less/ui/navigationSlider/styles.less [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+  }
+
+  &.vcv-ui-navigation-slider-dropdown-item--active,
+  &:hover {
+    background: #693bc5;
+  }
+}
+
+.rtl {
+
```
Comment (nikitahl) [inline public/sources/less/ui/navigationSlider/styles.less:178]: should this be empty?

File: public/sources/less/ui/navigationSlider/styles.less [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+    bottom: 0;
+    width: 18px;
+    background: #203251;
+    line-height: 47px;
+    text-align: center;
+  }
+
+  &:before {
+    left: 0;
+    //box-shadow: inset 20px 0 8px -10px rgba(32, 50, 81, 1);
```
Comment (nikitahl) [inline public/sources/less/ui/navigationSlider/styles.less:30]: are these comments needed?

File: public/sources/less/ui/navigationSlider/styles.less [hunk 4]
```diff
@@ -0,0 +1,179 @@
+.vcv-ui-navigation-slider-container {
+  display: flex;
+  justify-content: space-between;
+  position: relative;
+
+
+  &.vcv-ui-navigation-slider-container--shadow-left:before {
+    //content: '...';
```
Comment (nikitahl) [inline public/sources/less/ui/navigationSlider/styles.less:8]: are these comments needed?

File: public/sources/less/ui/navigationSlider/styles.less [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+    //content: '';
+  }
+
+  &.vcv-ui-navigation-slider-container--shadow-right:after  {
+    //content: '...';
```
Comment (nikitahl) [inline public/sources/less/ui/navigationSlider/styles.less:13]: are these comments needed?

File: public/sources/less/ui/navigationSlider/styles.less [hunk 6]
```diff
@@ -0,0 +1,179 @@
...
+  &:before {
+    left: 0;
+    //box-shadow: inset 20px 0 8px -10px rgba(32, 50, 81, 1);
+    //box-shadow: inset 10px 0 6px -5px rgba(32,50,81,1);
+    //background: linear-gradient(to right, #203251, transparent);
+  }
+
+  &:after {
+    right: @navigation-slider-control-width;
+    //box-shadow: inset -20px 0 8px -10px rgba(32, 50, 81, 1);
```
Comment (nikitahl) [inline public/sources/less/ui/navigationSlider/styles.less:37]: are these comments needed?

File: public/components/navigationSlider/navigationSlider.js (hunk 1 above)
Comment (konutis) [reply public/components/navigationSlider/navigationSlider.js]: Updated

**Review by nikitahl — CHANGES_REQUESTED** [review]

**Review by pavelthq — APPROVED** [review]

**Review by nikitahl — APPROVED** [review]

**Pull request merged**